modules/ROOT/pages/objc.adoc

```diff
@@ -1108,6 +1108,14 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
 +
 xref:index.adoc[Read more]
 
+*macOS Support*
+
+Support for macOS 10.9 and 10.10 is being deprecated in the 2.6 release and will be removed after two (non-maintenance) releases following the deprecation announcement. 
```

Comment:
@rajagp @ibsoln I find this deprecation notice confusing. It says that the deprecation is being made at 2.6, but this change is being merged into the 2.5 docs. Shouldn't this change be made in the 2.6 release notes?

And to be clear, if this notice is being made at 2.6, then in the following scenario I would expect it to mean the following:
* 2.6 (supported, deprecation announced)
* 2.7 (supported)
* 2.8 (supported)
* 2.9 (UNSUPPORTED)

Reply:
I took it to be 'advance' warning ... albeit retrospective. Or maybe it was pushed to wrong release in error. 

@rajagp @eric-schneider  Please advise change needed. Merge to 2.6 or re-word to make it clearer

Reply:
Oh dear...Its been a while since ticket was opened so I don't even recall! In any case, as of 2.5, the min supported version is 10.11. Which means 10.9 and 10.10 was deprecated in 2.5. Also note that macOS is only supported for testing and dev purposes...always.
2.5 (supported, deprecation announced). Yes, add to 2.5 release notes as well as Eric suggested 
2.6 (supported)
2.7 (supported)
2.8 (UNSUPPORTED)